src/App.js

```diff
@@ -15,7 +15,7 @@ function App() {
     setIsLoaded(false);
     setResults(null);
     fetch(
-      `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${process.env.REACT_APP_API_KEY}`
+      `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${process.env.REACT_APP_APIKEY}`
```

Comment:
Maybe add https://api.openweathermap.org/data/2.5/  to .env file as a REACT_APP_URL to make the fetch look a bit cleaner